lib/twitter_api.rb

```diff
@@ -95,28 +95,17 @@ def twitter_credentials_missing?
     protected
 
     def link_handles_in(text)
-      text.scan(/(?:^|\s)@(\w+)/).flatten.uniq.each do |handle|
-        text.gsub!(/(?:^|\s)@#{handle}/, [
-          " <a href='https://twitter.com/#{handle}' target='_blank'>",
-            "@#{handle}",
-          "</a>"
-        ].join)
-      end
-
-      text.strip
+      text.gsub(/(?:^|\s)@(\w+)/) do |match|
```

Comment:
That group isn't used, is it?

```suggestion
      text.gsub(/(?:^|\s)@\w+/) do |match|
```

Reply:
Alternatively it could be used below: `match[1].strip` 🤷

Reply:
You are correct, the group is unused and unfortunately `match` is a `String` object and does not contain the captured groups. :(